setup.py

```diff
@@ -37,16 +37,15 @@
         'Issue Tracker': 'https://github.com/agrc/python/issues',
     },
     keywords=['gis'],
-    install_requires=[
-        # 'package==1.0.*'
-    ],
+    install_requires=['arcgis'],
```

Comment:
**quibble**: this should have the ==2 or >1 or something so you get the correct package

Reply:
I wanted to make sure this installed on as broad a collection of arcgis pro installations as possible without having to do anything but pip install, so I left it bare. I'll change it to >1.7 or something like that.